02-esc/src/mixins.scss

```diff
@@ -5,7 +5,12 @@
 }
 
 @mixin responsive-ctas {
+    display: -webkit-box;
+    display: -ms-flexbox;
```

Comment:
Not sure what the reasoning is behind this, but I would recommend against using vendor prefixed properties. There was a time when these were important, but nowadays with "evergreen" browsers, they are increasingly irrelevant.

Reply:
The reasoning is I forgot to turn off my "AutoPrefixer" VS Code extension. 😁 Lesson learned for future group assignments.